CHANGELOG.md

```diff
@@ -35,6 +36,7 @@
  - Zone
  - ZoneCollection
  - ResourceBlock
+ - StorageResourceBlock
```

Comment:
I believe you can remove this `StorageResourceBlock` as it is not a Redfish resource